src/WebJobs.Script/Host/FunctionMetadataProvider.cs

```diff
@@ -32,12 +32,12 @@ public FunctionMetadataProvider(ILogger<FunctionMetadataProvider> logger, IWorke
 
         public ImmutableDictionary<string, ImmutableArray<string>> FunctionErrors { get; private set; }
 
-        public async Task<ImmutableArray<FunctionMetadata>> GetFunctionMetadataAsync(IEnumerable<RpcWorkerConfig> workerConfigs, IEnvironment environment, bool forceRefresh = false)
+        public async Task<ImmutableArray<FunctionMetadata>> GetFunctionMetadataAsync(IEnumerable<RpcWorkerConfig> workerConfigs, bool forceRefresh = false)
```

Comment:
Just double checking that this is just clean up/tidy work since environment is never actually used?

Reply:
That is correct.